ScoreSaber/UI/Leaderboard/ScoreSaberLeaderboardViewController.cs

```diff
@@ -446,62 +621,214 @@ private async Task StartReplay(ScoreMap score) {
             _replayDownloading = false;
         }
 
+        private void SetClickersOff() {
+            foreach (var holder in _cellClickingHolders) {
+                var x = holder.cellClickerImage.gameObject.GetComponent<CellClicker>();
+                if (x != null) {
+                    x.clickable = false;
+                }
+            }
+        }
+
+        private bool obtainedAnchor = false;
+        private Vector2 normalAnchor = Vector2.zero;
+
+        void PatchLeaderboardTableView(LeaderboardTableView tableView, ScoreMap[] leaderboardInfo, CancellationToken cancellationToken) {
```

Comment:
this isn't really doing any patching, so a different name would be more appropriate